<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>15</red>
        <green>128</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>15</red>
        <green>128</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>15</red>
        <green>128</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>15</red>
        <green>128</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="player_img_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ai_img_lb">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>220</width>
      <height>320</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Computer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="player_name_lb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Hero</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="stat_lb">
       <property name="font">
        <font>
         <family>Futura</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Futura</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Winner:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="win_lb">
       <property name="font">
        <font>
         <family>Futura</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="victory_lb">
       <property name="font">
        <font>
         <family>Futura</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ai_name_lb">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Hero</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="player_intel_btn">
         <property name="text">
          <string>Intelligence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_str_btn">
         <property name="text">
          <string>Strength</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_spd_btn">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_dura_btn">
         <property name="text">
          <string>Durability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_pwr_btn">
         <property name="text">
          <string>Power</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_combat_btn">
         <property name="text">
          <string>Combat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="player_intel_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_str_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_spd_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_dura_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_pwr_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_combat_lb">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>160</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="lbl5">
         <property name="text">
          <string>Intelligence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player_str_btn_2">
         <property name="text">
          <string>Strength</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lbl4">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lbl3">
         <property name="text">
          <string>Durability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lbl2">
         <property name="text">
          <string>Power</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lbl1">
         <property name="text">
          <string>Combat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="ai_intel_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ai_str_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ai_spd_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ai_dura_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ai_pwr_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ai_combat_lb_2">
         <property name="text">
          <string>###</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
